vent/menu.py

```diff
@@ -6,6 +6,9 @@
 import npyscreen
 import subprocess
 
+from helpers.meta import Version
```

Comment:
If the user doesn't install `vent` as a package prior to running `vent` through `python menu.py` that will break I believe.